cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -80,7 +80,7 @@ jenkins_java_extra_options: []
 # comfortably without having to re-login.
 session_timeout: 480
 # Command line arguments for Jenkins
-jenkins_args:
+jenkins_args: "--sessionTimeout={{ session_timeout }}"
```

Comment:
Currently this would allow a user to accidentally override their `session_timeout` value unintentionally by setting a custom value for `jenkins_args`. It might be better to construct something like this in the `vars/main.yml` file (placing it in `vars/main.yml` instead of here in `defaults/main.yml` would be important, as values in `vars` are difficult to override in the precedence order and should be used by values that are not intended to be user configurable):

```yaml
_jenkins_args: --sessionTimeout={{ session_timeout }} {{ jenkins_args }}
```

Then you can change the places that currently reference `{{ jenkins_args }}` in templates to instead reference `{{ _jenkins_args }}`. That prevents the user from accidentally overriding the behavior of `session_timeout` without realizing it.

Reply:
good point, thanks.  I will do that.